packages/redux/src/utils.ts

```diff
@@ -26,10 +32,16 @@ export function createReduxStore(config: {
     ? (window['__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'] || compose)
     : compose;
 
+  // if config has namespaceCrafterState set to true, combines crafter reducers into namespace, plus config reducers
+  // (if available), otherwise, combines crafter reducers directly on root of state.
+  const reducer = (config.namespaceCrafterState)
+    ? <ReducersMapObject<CrafterNamespacedState, AnyAction>>{ [config.namespace]: combineReducers(allReducers) }
+    : <ReducersMapObject<CrafterState, AnyAction>>allReducers;
+
   return createStore(
     config.reducerMixin
-      ? combineReducers({ ...allReducers, ...config.reducerMixin })
-      : combineReducers(allReducers),
+      ? combineReducers({ ...reducer, ...config.reducerMixin })
+      : combineReducers(reducer),
```

Comment:
Find out how to apply multiple middleware and allow for that to be supplied too. for example, if people want to use redux-thunk instead of redux-observable for their own app.